pkgs/development/tools/ocaml/dune/default.nix

```diff
@@ -2,10 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "dune";
-  version = "1.10.0";
+  version = "1.11.0";
   src = fetchurl {
-    url = "https://github.com/ocaml/dune/releases/download/${version}/dune-${version}.tbz";
-    sha256 = "15fx9rg16g7ig43rg4sdq0wp0br5h1mjxxgv8b15s317vqlfc5pd";
+    url = "https://github.com/ocaml/dune/releases/download/${version}/dune-build-info-${version}.tbz";
```

Comment:
This change looks wrong.

Reply:
It seems that the hash is wrong now.

Reply:
Thanks, I've update the hash.
About the .tbz name seems to be related to https://github.com/ocaml/dune/pull/2224, and it is the same that dune declares in the opam archive https://github.com/ocaml/opam-repository/pull/14567/files#diff-a081995843fd7c2880ab94240401e49bR48